frameit/frames_generator/README.md

```diff
@@ -0,0 +1,12 @@
+# Prepare and upload new device frames
+
+Since October 2016 we host the device frame resources since Apple removed lots of old device frame images.
```

Comment:
```
Starting in October 2016 we are now using the (Facebook frameset)[http://facebook.design/devices] and hosting the device frame resources since Apple sometimes has removed / changed some of the old images making framing a moving target.

To run the wizard to prepare new device assets, just run `rake` in this directory.

You will need to run this wizard every time you want update the device assets

This will generate 2 directories:

- The timestamp based one which is used if the user specifies a particular version in their Framefile
- The the most recent one which will be used by default
```